packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.ts

```diff
@@ -317,7 +328,7 @@ export class HorizontalBarChart extends FASTElement {
       .attr('height', 12)
       .attr('width', 100 + '%')
       .attr('class', 'svg-chart')
-      .attr('aria-label', data?.chartTitle ? data?.chartTitle : '')
+      .attr('aria-label', data?.barTitle ? data?.barTitle : '')
```

Comment:
lets use a default aria label

Reply:
title for each row should always be present. should I set it regardless?

Reply:
very corner case. But lets still set it.

Reply:
done